### PR TITLE
Enable branding of snaps

### PR DESCRIPTION
**Pull request by kelvin-chappell**
This change adds branding to any `FaciaContent`.

I've tested it in frontend on my machine, by pressing fronts in draft and live mode, and the results are as expected.

### CODE REVIEW

**Comment by kelvin-chappell** [comment]
[Add Paid styling to snap for GLabs](https://trello.com/c/jYqp5zya/135-add-paid-styling-to-snap-for-glabs)

**Comment by kelvin-chappell** [comment]
[Add Paid styling to snap for GLabs](https://trello.com/c/jYqp5zya/135-add-paid-styling-to-snap-for-glabs)

File: fapi-client/src/main/scala/com/gu/facia/api/models/collection.scala [hunk 1]
```diff
@@ -88,23 +101,38 @@ object Collection {
...
+        .filter(_.safeMeta.snapType.contains("latest"))
         .flatMap(snap => snap.meta.flatMap(_.snapUri).map(uri => snap.id ->uri))
         .toMap)
 
   def liveLatestSnapsRequestFor(collection: Collection): LatestSnapsRequest =
     LatestSnapsRequest(
       collection.live
       .filter(_.isSnap)
-      .filter(_.safeMeta.snapType == Some("latest"))
+      .filter(_.safeMeta.snapType.contains("latest"))
```
Comment (janua) [inline fapi-client/src/main/scala/com/gu/facia/api/models/collection.scala]: Are we sure all clients are on Scala `2.11`?

File: fapi-client/src/main/scala/com/gu/facia/api/models/curatedcontent.scala [hunk 2]
```diff
@@ -58,35 +59,47 @@ object FaciaImage {
 
 }
 
-sealed trait FaciaContent
+sealed trait BrandableContent {
+  def brandingByEdition: BrandingByEdition = Map.empty
+}
+
+sealed trait FaciaContent extends BrandableContent
```
Comment (janua) [inline fapi-client/src/main/scala/com/gu/facia/api/models/curatedcontent.scala]: Why the double sealed trait dependency?

File: fapi-client/src/main/scala/com/gu/facia/api/models/collection.scala (hunk 1 above)
Comment (kelvin-chappell) [inline fapi-client/src/main/scala/com/gu/facia/api/models/collection.scala]: It looks like only a 2.11 version of the artefact gets released.

File: fapi-client/src/main/scala/com/gu/facia/api/models/curatedcontent.scala (hunk 2 above)
Comment (kelvin-chappell) [inline fapi-client/src/main/scala/com/gu/facia/api/models/curatedcontent.scala]: It doesn't seem to do any harm but I can unseal `FaciaContent` if you prefer?

**Comment by janua** [comment]
:+1:

**Pull request merged**